### PR TITLE
document how to skip a Gradle task

### PR DESCRIPTION
**Pull request by denisbedin**
https://docs.microej.com/en/feature-microej-sdk-6-skip-task/SDK6UserGuide/howtos/gradleSkipTask.html

### CODE REVIEW

File: SDK6UserGuide/howtos/gradleSkipTask.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+   gradle build -x test
+
+  The task is skipped for this execution only.
+
+- Exclude the task in the build script of the project ::
+
+   project.gradle.startParameter.excludedTaskNames.add("test")
+
+  The task is never executed.
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/howtos/gradleSkipTask.rst:21]: Isn't there a difference between excluding a task with `-x` and with `project.gradle.startParameter.excludedTaskNames`. I had in mind that when you use `-x` it does exclude the tasks on which the excluded task is dependent. Am I wrong?

**Review by tdelhomenie-microej — CHANGES_REQUESTED** [review]

File: SDK6UserGuide/howtos/gradleSkipTask.rst (hunk 1 above)
Comment (denisbedin) [reply SDK6UserGuide/howtos/gradleSkipTask.rst:21]: I didn't notice this difference between the 2 ways to exclude a task. I tried with the `test` task, that depends on the `testClasses` task, and if I exclude the `test` task, the `testClasses` task is also excluded regardless of the way used to exclude the task (`-x` or  `project.gradle.startParameter.excludedTaskNames`).

Comment (tdelhomenie-microej) [reply SDK6UserGuide/howtos/gradleSkipTask.rst:21]: Right, I don't remember I had this assertion in mind.

**Review by tdelhomenie-microej — APPROVED** [review]

File: SDK6UserGuide/howtos/gradleSkipTask.rst [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+When one of these two ways is used, not only the task but also all the tasks on which it depends are skipped.
+For example, if you choose to skip the ``test`` task, all the tasks which are used to produce 
+the test runtime classpath are also skipped.
+
+The next section describes how you can skip a task, but still execute the tasks on which it depends.
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/howtos/gradleSkipTask.rst:27]: I would remove this line, it is redundant with the next one.

Comment (denisbedin) [reply SDK6UserGuide/howtos/gradleSkipTask.rst]: Done.

**Pull request merged**